### PR TITLE
membership view services via a form

### PR DESCRIPTION
**Pull request by tswfi**
* added services form in apply membership view
* this keeps the selections with errors etc
* the datetime widget didn't support localization correctly, fix that also

### CODE REVIEW

**Comment by tswfi** [comment]
I don't understand why the test fails like that...

**Comment by tswfi** [comment]
oh, it doesn't understand if you have a totally empty db...

**Comment by tswfi** [comment]
aaah, because it tries to build things in the RegistrationForm..

File: users/models.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ class CustomUser(AbstractUser):
     birthday = models.DateField(
         blank=False,
         verbose_name=_('Birthday'),
-        help_text=_('Format: DD.MM.YYYY'),
+        # help_text=_('Format: DD.MM.YYYY'),
```
Comment (vranki) [inline users/models.py]: Remove comment, unless this needs to be returned.

Comment (tswfi) [reply users/models.py]: https://github.com/TampereHacklab/mulysa/pull/109/commits/039107ef48761d2457eb3511ec85b4271000b27f

**Pull request merged**